src/index.js

```diff
@@ -1,6 +1,19 @@
 import React from 'react'
 import ReactDOM from 'react-dom'
+import {Route, BrowserRouter as Router, Switch} from 'react-router-dom'
 import './index.css'
 import Page from './components/Page/Page'
+import Notfound from './components/Notfound/Notfound'
 
-ReactDOM.render(<Page />, document.getElementById('root'))
+function Routing() {
+    return (
+        <Router>
+            <Switch>
+                <Route exact path="/" component={Page}/>
+                <Route component={Notfound}/>
```

Comment:
I think that this should still define a path match so that we're clear about what it's doing